index.html

```diff
@@ -15,7 +14,78 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
+
   <body>
+    <header>
+      <h1>Full Stack Web Developer</h1>
+      <p>(Lecture Notes)</p>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="#">JavaScript</a></li>
+        <li><a href="#html">HTML</a></li>
+        <li><a href="#css">CSS</a></li>
+        <li><a href="#git">Git/Github</a></li>
+        <li><a href="#">React</a></li>
+      </ul>
+    </nav>
+<main>
+    <article id="html">
+      <h2>What is HTML? </h2>
+      <p>HTML (HyperText Markup Language) is the language used to create documents that are displayed on web browsers. It gives structure and meaning to content by using tags such as &lt h1&gt and &ltp&gt to describe headings and paragraphs of text.</p>
+      <h4>HTML Anatomy</h4>
+      <img src="https://syllabus.codeyourfuture.io/13e3d224c01fe17853373d2c322a74e7.png">
+      <p>An HTML element is made up of <strong>tags</strong>, <strong>attributes</strong>, and <strong>content</strong></p>
+        <ul>
+          <li>A <strong>tags</strong> is the HTML element name enclosed in angle brackets</li>
+          <li>An <strong>attributes</strong> comes after the HTML element name within the angle brackets</li>
+          <li>The <strong>content</strong> is the part of the code in between the opening and closing tags</li>
+        </ul>
+        <a href="https://www.w3schools.com/html/default.asp">Read More</a>
+      </article>
```

Comment:
If you want to emphasise a piece of text the recommended tag is the `em` tag. `strong` is used to denote that whatever is inside the tag is important - e.g. if it was a warning message.